lib/grover/js/processor.cjs

```diff
@@ -16,35 +16,46 @@ const path = require('path');
 
 const _processPage = (async (convertAction, urlOrHtml, options) => {
   let browser, errors = [], tmpDir;
-  try {
-    tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'grover-'));
-
-    const launchParams = {
-      args: process.env.GROVER_NO_SANDBOX === 'true' ? ['--no-sandbox', '--disable-setuid-sandbox'] : [],
-      userDataDir: tmpDir
-    };
-
-    // Configure puppeteer debugging options
-    const debug = options.debug; delete options.debug;
-    if (typeof debug === 'object' && !!debug) {
-      if (debug.headless !== undefined) { launchParams.headless = debug.headless; }
-      if (debug.devtools !== undefined) { launchParams.devtools = debug.devtools; }
-    }
 
-    // Configure additional launch arguments
-    const args = options.launchArgs; delete options.launchArgs;
-    if (Array.isArray(args)) {
-      launchParams.args = launchParams.args.concat(args);
-    }
+  // Configure puppeteer debugging options
+  const debug = options.debug; delete options.debug;
+  const remoteConnect = typeof options.browserWsEndpoint === "string";
+  try {
+    if (remoteConnect) {
+      const connectParams = {
+        browserWSEndpoint: options.browserWsEndpoint,
+      };
```

Comment:
i'd recommend the pattern:
```suggestion
    const browserWsEndpoint = options.browserWsEndpoint; delete options.browserWsEndpoint;
    if (typeof browserWsEndpoint === "string") {
      const connectParams = {
        browserWSEndpoint: browserWsEndpoint,
      };
```
The main reason is because the options are simply passed through verbatim to the convert action (`pdf`, `content` or `screenshot`). Safer to remove these sorts of custom options as we check/use them to prevent any sort of unexpected behaviour elsewhere!